frontend/src/concepts/pipelines/content/createRun/contentSections/ParamsSection/RadioInputParam.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Flex, Radio } from '@patternfly/react-core';
+import { InputParamProps } from './types';
+
+enum RadioValue {
+  True = 'true',
+  False = 'false',
+}
+
+export const RadioInputParam: React.FC<InputParamProps> = ({
+  id,
+  value: initialValue,
+  onChange,
+}) => {
+  const [radioValue, setRadioValue] = React.useState(initialValue);
```

Comment:
why are you duplicating the state here? You can probabaly just use the value as the value instead of pushing that into the radioValue state